build.sbt

```diff
@@ -24,19 +24,34 @@ import uk.gov.hmrc.versioning.SbtGitVersioning
 
 lazy val library = (project in file("."))
   .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
+  .settings(PlayCrossCompilation.playCrossCompilationSettings)
   .settings(
     scalaVersion := "2.12.10",
     name := "play-json-union-formatter",
     majorVersion := 1,
     makePublicallyAvailableOnBintray := true,
     targetJvm := "jvm-1.8",
-    libraryDependencies ++= Seq(
-      "org.scalatest"     %% "scalatest" % "3.0.8"  % "test",
-      "uk.gov.hmrc"       %% "hmrctest"  % "3.9.0-play-26"  % "test",
-      "com.typesafe.play" %% "play-json" % "2.6.14"
-    ),
+    libraryDependencies ++= deps,
     resolvers := Seq(
       Resolver.bintrayRepo("hmrc", "releases")
     )
   )
 
+val testDepsShared: Seq[ModuleID] = Seq(
+  "org.scalatest" %% "scalatest" % "3.0.8" % "test",
+  "org.pegdown"   %  "pegdown"   % "1.6.0" % "test",
```

Comment:
Do we need this library? It looks like it wasn't there before. Unless it was a transitive dependency of hmrctest or something like that?

Reply:
Yes, unfortunately without it we get a `NoClassDefFoundError` when trying to run the tests as I think this library is used for rendering HTML reports